README.md

```diff
@@ -408,6 +409,131 @@ options = StreamOptions(
 client.stream(fql('Product.all().toStream()'), options)
 ```
 
+## Change Feeds
+
+<!-- TODO: turn "Changefeeds" into a link when available. -->
+
+The driver supports Changefeeds.
+
+### Start a changefeed
+
+Changefeeds work by interactively polling from a [Fauna Stream](https://docs.fauna.com/fauna/current/learn/streaming). 
+To get a stream token, append ``toStream()`` or ``changesOn()`` to a set from a
+[supported source](https://docs.fauna.com/fauna/current/reference/streaming_reference/#supported-sources).
+
+To create a changefeed, pass the stream token to ``change_feed()`` method:
+
+```python
+  from fauna import fql
+  from fauna.client import Client
+
+  client = Client()
+
+  response = client.query(fql('''
+  let set = Product.all()
+  {
+    initialPage: set.pageSize(10),
+    streamToken: set.toStream()
+  }
+  '''))
+
+  initialPage = response.data['initialPage']
+  streamToken = response.data['streamToken']
+
+  client.change_feed(streamToken)
+```
+
+You can also pass a query that produces a stream token directly to
+``change_feed()``:
+
+```python
+  query = fql('Product.all().changesOn(.price, .quantity)')
+
+  client.change_feed(query)
+```
+
+### Iterate on a changefeed
+
+``change_feed()`` returns an iterator that emits pates of events on every
+iteration. You can use a generator expression to iterate through the pages:
+
+```python
+query = fql('Product.all().changesOn(.price, .quantity)')
+
+with client.change_feed(query) as feed:
+  for page in feed:
+    print('Page stats: ', page.stats)
+
+    for event in page.events:
+      eventType = event['type']
+      if (eventType == 'add'):
+        print('Add event: ', event)
+        ## ...
+      elif (eventType == 'update'):
+        print('Update event: ', event)
+        ## ...
+      elif (eventType == 'remove'):
+        print('Remove event: ', event)
+        ## ...
+```
+
+Alternatively, you can iterate through events instead of pages with
+``flatten()``:
+
+```python
+query = fql('Product.all().changesOn(.price, .quantity)')
+
+with client.change_feed(query) as feed:
+  for event in feed.flatten():
+    eventType = event['type']
+    ## ...
+```
+
+The changefeed iterator will stop once there are no more events to poll.
```

Comment:
```suggestion
The changefeed iterator stops when there are no more events to poll.
```

We should probably mention that you can later poll the stream for new events by specifying the last event's `cursor` or a `start_ts`.